backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/invoice/C_Invoice_StepDef.java

```diff
@@ -177,4 +277,11 @@ private void validateInvoice(@NonNull final I_C_Invoice invoice, @NonNull final
 		assertThat(paymentTermId).isNotNull();
 		assertThat(invoice.getC_PaymentTerm_ID()).isEqualTo(paymentTermId.getRepoId());
 	}
+
+	private void validateInvoiceSalesRep(@NonNull final I_C_Invoice invoice, @NonNull final Map<String, String> row)
+	{
+		final int expectedSalesRep_ID = DataTableUtil.extractIntOrMinusOneForColumnName(row, "OPT." + I_C_Order.COLUMNNAME_SalesRep_ID);
+
+		assertThat(invoice.getSalesRep_ID()).isEqualTo(expectedSalesRep_ID);
```

Comment:
as mentioned above, the check should be done in the existing `validateInvoice` method.